api/DOMStringMap.json

```diff
@@ -6,10 +6,10 @@
         "spec_url": "https://html.spec.whatwg.org/multipage/dom.html#domstringmap",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "7"
```

Comment:
`DOMStringMap` is the type of `element.dataset`, and that was added in Chrome 8 per our data. It's possible the interface was accidentally shipped before the thing using it, so I won't check further. If `element.dataset` was actually shipped in Chrome 7 too, we should eventually discover that using the collector as well.